src/components/form/PartnerGoogleAccountSelect.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {makeStyles} from '@material-ui/core/styles';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+
+const useStyles = makeStyles((theme) => ({
+  formControl: {
+    backgroundColor: '#ffffff',
+    color: '#457b9d',
+    minWidth: 120,
+    textAlign: 'left',
+    width: '100%',
+  },
+
+  input: {
+    color: '#1d3557',
+    '&:before': {
+      color: '#1d3557',
+    },
+    '&:after': {
+      color: '#1d3557',
+    },
```

Comment:
I had a problem with changing label color, do you know how to do it with Material UI? I mean this "focus" mode.

Reply:
I think it's `.Mui-focused`.

I went to https://material-ui.com/components/text-fields/, activated the source code tab in the dev tools (`F12`) and selected on of the fields. That class got added.

Reply:
Done :)